helixbass/tsc-rust
# Parse variable declaration with type annotation

In this PR:
- parse a `const` variable declaration with `: number` type annotation

To test:
If you have a file named eg `tmp.tsx` in the repo root directory whose contents are `const a: number = true` and run `cargo run tmp.tsx` you should see debug-logging of the `SourceFile` with a `VariableStatement` with a `VariableDeclarationList` containing a single `VariableDeclaration` whose `name` is the `Identifier` `"a"`, `initializer` is a `TrueKeyword` and `type_` is a `NumberKeyword` `KeywordTypeNode` and then it should panic during binding
